rest/taskrouter/workflows/list/post/example-1/example-1.4.x.php

```diff
@@ -0,0 +1,47 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library
+require_once('/path/to/twilio-php/Services/Twilio/WorkflowConfiguration.php'); // Loads the non-autoload class (due to class naming with this version)
```

Comment:
I have tested this snippet with v4.11. Autoload still seems to be broken. This issue https://github.com/twilio/twilio-php/issues/312 was closed because https://github.com/twilio/twilio-php/pull/314 was merge but I don't see it merged. Adding the second require does fix the snippet, but testing only works with autoload.